README_fr-FR.md

```diff
@@ -0,0 +1,443 @@
+
+Pour maximiser ces résultats, nous pouvons utiliser certaines bonnes pratiques et des standards, décrits dans la prochaine section.
+
+## Bons Usages
+
+Il y a quelques bons usages recueillit par l'experience, des articles sur internet, et d'autres guides. Si vous en avez d'autres (ou n'êtes pas d'accord avec certains), soyez libres d'ouvrir une demande de Pull et de contribuer à l'amélioration du guide.
+
+### Utilisez l'impératif
+
+```
+# Bon
+Use InventoryBackendPool to retrieve inventory backend
+```
+
+```
+# Mauvais 
+Used InventoryBackendPool to retrieve inventory backend
+```
+
+_Pourquoi l'impératif ?__
+
+Un message de commit décrit ce que le changement **fait**, ces effets, pas ce qui à été fait.
+
+[Cet excelent article de Chris Beams](https://chris.beams.io/posts/git-commit/)
+Nous donne une phrase simple qui peut être utilisée pour mieux écrire les messages de commit à l'impératif.
+
+```
+If applied, this commit will <commit message>
+```
+
+Examples:
+
+```
+# Bon
+If applied, this commit will use InventoryBackendPool to retrieve inventory backend
+```
+
+```
+# Mauvais
+If applied, this commit will used InventoryBackendPool to retrieve inventory backend
+```
+
+### Capitalize the first letter
+
+```
+# Bon
+Add `use` method to Credit model
+```
+
+```
+# Mauvais
+add `use` method to Credit model
+```
+
+The reason that the first letter should be capitalized is to follow the grammar rule of using capital letters at the beginning of sentences.
+
+The use of this practice may vary from person to person, team to team, or even from language to language.
+Capitalized or not, an important point is to stick to a single standard and follow it.
+
+### Try to communicate what the change does without having to look at the source code
+
+```
+# Bon
+Add `use` method to Credit model
+
+```
+
+```
+# Mauvais
+Add `use` method
+```
+
+```
+# Bon
+Increase left padding between textbox and layout frame
+```
+
+```
+# Mauvais
+Adjust css
+```
+
+It is useful in many scenarios (e.g. multiple commits, several changes and refactors) to help reviewers understand what the committer was thinking.
+
+### Use the message body to explain "why", "for what", "how" and additional details
+
+```
+# Bon
+Fix method name of InventoryBackend child classes
+
+Classes derived from InventoryBackend were not
+respecting the base class interface.
+
+It worked because cart was calling the backend implementation
+incorrectly.
+```
+
+```
+# Bon
+Serialize and deserialize credits to json in Cart
+
+Convert the Credit instances to dict for two main reasons:
+
+  - Pickle relies on file path for classes and we do not want to break up
+    everything if a refactor is needed
+  - Dict and built-in types are picklable by default
+```
+
+```
+# Bon
+Add `use` method to Credit
+
+Change from namedtuple to class because we need to
+setup a new attribute (in_use_amount) with a new value
+```
+
+The subject and the body of the messages are separated by a blank line.
```

Comment:
`The subject and the body of the messages are separated by a blank line.` > `Le sujet et le corps des messages sont séparés par une ligne vide.`